src/app/app.component.css

```diff
@@ -31,6 +31,12 @@ mat-divider {
   margin: 16px;
 }
 
+@media screen and (max-width: 400px) {
+  .content {
+    margin: 0;
+  }
+}
+
```

Comment:
what is this for?

Reply:
this is the style that defines no margins at a maximum screen width of 400